Makefile

```diff
@@ -18,16 +18,20 @@ DOWNLOADFILES := $(BIN_DOWNLOAD_FILEPATH) $(LOG_DOWNLOAD_FILEPATH)
 
 help:
 	@echo 'Cleaning targets:'
-	@echo '  clean                      - clean the cache artifacts'
+	@echo '  clean                      - clean the cache artifacts and generated files'
 	@echo 'Generic targets:'
 	@echo '* all                        - build solidity code. To build from a clean clone, run: make submodules all'
 	@echo '  build                      - build solidity code'
-	@echo '  dep                        - install npm packages'
-	@echo '  depclean                   - remove npm packages'
-	@echo '  deploy                     - deploy to local node'
-	@echo '  generate                   - generate solidity code from cpp and template'
-	@echo '  test                       - test both binary files and log files'
-	@echo '  coverage                   - generate coverage report for html view'
+	@echo '  generate-all               - generate all solidity code'
```

Comment:
The CI needs a check to know whether a regeneration is needed, because the emulator sources may change and we will may forget to manually regenerate the sources in `ready_src`. But lets leave this as an enhancement for later.

Reply:
Yeah @mpernambuco mentioned this too. Maybe use `shasum` to check?